ldap-client.go

```diff
@@ -254,6 +256,50 @@ func (lc *LDAPClient) GetAllGroupsByName(groupName string) ([]LdapGroup, error)
 	return groups, nil
 }
 
+// GetAllGroupsWithMembersByName returns list of groups matching a name.
+func (lc *LDAPClient) GetAllGroupsWithMembersByName(groupsDN string, groupCN []string) ([]*LdapGroup, error) {
+	err := lc.Connect()
+	if err != nil {
+		return nil, err
+	}
+
+	dn := lc.BindDN
+	if groupsDN != "" {
```

Comment:
Why not configure the LDAPClient with groupsDN as the baseDN?

Mainly to use the single source of truth regarding the LDAPClient which one he uses as to where to bind to in the LDAP.

WDYT? @yvgenycyolo